### PR TITLE
[DOCFIX] [branch-1.3] Improve configuration to run Alluxio with HDFS

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-HDFS.md [hunk 1]
```diff
@@ -45,17 +45,20 @@ For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_H
...
+Alternatively, you can also create the configuration file from the template and set the contents manually.
 
 {% include Common-Commands/copy-alluxio-env.md %}
 
-Then edit `alluxio-env.sh` file to set the under storage address to the HDFS namenode address
-(e.g., `hdfs://localhost:9000` if you are running the HDFS namenode locally with default port).
+Then edit `conf/alluxio-site.properties` file to set the under storage address to the HDFS namenode address
+and the HDFS directory you want to mount to Alluxio. For example, the under storage address can be
+`hdfs://localhost:9000` if you are running the HDFS namenode locally with default port and mapping HDFS root directory to Alluxio,
+`hdfs://localhost:9000/alluxio/data` if only the HDFS directory `/alluxio/data` is mapped to Alluxio.
```
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: nit: or `hdfs://localhost:9000/alluxio/data` if only ...

Comment (apc999) [inline docs/en/Configuring-Alluxio-with-HDFS.md]: fixed

**Comment by apc999** [comment]
@ifcharming PTAL

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**